style.css

```diff
@@ -38,4 +38,36 @@ svg {
 
 .segment.leaf.play.stuff {
   fill: yellow;
-}
\ No newline at end of file
+}
+
+.circletree .leaf {
+  opacity: 0;
+}
+
+.circletree g {
+  opacity: .3;
+}
+
+.circletree g.primary {
+  opacity: 1;
+}
+
+.circletree g.primary g {
+  opacity: 1;
+}
+
+.circletree g.primary .leaf {
+  opacity: 1;
+}
+
+.circletree g.active {
+  opacity: 1;
+}
+
+g .circletree g.active:not(.primary) {
```

Comment:
This is not needed, just an artifact as I initially wrote this in less, and the compiler added this. I'll remove.